.github/workflows/integrations.yml

```diff
@@ -147,17 +147,33 @@ jobs:
         env:
           CMAKE_TOOLCHAIN_FILE: ${{ github.workspace }}/.conan/conan_paths.cmake
 
+      - name: Checkout TraitGen
+        uses: actions/checkout@v3
+        with:
+          repository: OpenAssetIO/OpenAssetIO-TraitGen
+          path: external/TraitGen
+
+      - name: Install TraitGen
+        run : |
+          python -m pip install external/TraitGen
+
       - name: Checkout OpenAssetIO-Test-CMake
         uses: actions/checkout@v3
         with:
-          repository: OpenAssetIO/OpenAssetIO-Test-CMake
+          repository: elliotcmorris/OpenAssetIO-Test-CMake
```

Comment:
[fix] Just to ensure this isn't forgotten.

Reply:
Aye i'm just gonna drop that commit.